packages/router/README.md

```diff
@@ -170,3 +170,52 @@ const SomePage = () => {
   ...
 }
 ```
+
+## Code-splitting
+
+By default, RR (when used in a Redwood app) will code-split on every Page, creating a separate lazy-loaded webpack bundle for each. When navigating from page to page, RR will wait until the new Page module is loaded before re-rendering, thus preventing the "white-flash" effect.
+
+## Not code splitting
+
+If you'd like to override the default lazy-loading behavior and include certain Pages in the main webpack bundle, you can simply add the import statement to the `Routes.js` file:
+
+```js
+// Routes.js
+
+import HomePage from 'src/pages/HomePage'
+```
+
+Redwood will detect your explicit import and refrain from splitting that page into a separate bundle. Be careful with this feature, as you can easily bloat the size of your main bundle to the point where your initial page load time becomes unacceptable.
+
+## PageLoadingContext
+
+Because lazily-loaded pages can take a non-negligible amount of time to load (depending on bundle size and network connection), you may want to show a loading indicator to signal to the user that something is happening after they click a link. RR makes this really easy with `PageLoadingContext`:
+
+```js
+// SomeLayout.js
+
+import { PageLoadingContext } from '@redwoodjs/router'
+
+const SomeLayout = () => (
+  <div>
+    <PageLoadingContext.Consumer>
+      {(context) => {
```

Comment:
Instead of calling this `context` (even though it is) it might be easier to understand if the variable is called `loading` in this example?

Do you think you might want to make additional values available in the Page Loading Context in the future? It might be nice to return an object here, and destructure `{ loading }` to make this future-proof.